frontend-project/src/services/cases.js

```diff
@@ -0,0 +1,5 @@
+import request from '@/utils/request';
+
+export async function queryInstitutions() {
+    return request('/api/institutions');
```

Comment:
@kuskoman , tutaj chyba raczej chcemy iść w kierunku użycia SDK jak w przypadku ```tags.js```?

Reply:
@ad-m tak, pasuje żeby było to ujednolicone. póki co idziemy wszyscy w kierunku SDK, więc proponuję przy tym pozostać.

Reply:
becnie mamy 600+ instytucji w small_eod, a w pokrewnym systemie (Fedrowanie) Stowarzyszenie ma zarejestrowany kontakt z 5k+ instytucje. Myślę,że potrzebujemy tekstowe przeszukiwać, zamiast pobrać je wszystkie.

Reply:
Obecnie na produkcji tagów mamy 79, a użytkowników 34, stąd się wcześniej nie odzywałem.